test/e2e/e2e_test.go

```diff
@@ -258,7 +258,7 @@ func (o *options) DefaultClusterOptions(t *testing.T) core.CreateOptions {
 		ControlPlaneOperatorImage:        o.configurableClusterOptions.ControlPlaneOperatorImage,
 		ExternalDNSDomain:                o.configurableClusterOptions.ExternalDNSDomain,
 		AWSPlatform: core.AWSPlatformOptions{
-			InstanceType:       "m5.large",
+			InstanceType:       "m5.xlarge",
```

Comment:
why is this needed now and it was not before?

Reply:
I'm still testing but the example HC we create now manages 1-N NodePools, so eventually this HC will need some boost. The plan is to check if the older one is enough to handle all 4 tests and if not we can continue the discussion

Reply:
> HC we create now manages 1-N NodePools,
I'm not sure how this relates to increase the size of the the NodePool instances

Reply:
Woah you're right, brain fart... I will update the PR soon (after the Tests execution in local)